CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - GHA to verify checklist items completion in PR descriptions ([#10800](https://github.com/opensearch-project/OpenSearch/pull/10800))
 - [Remote cluster state] Restore cluster state version during remote state auto restore ([#10853](https://github.com/opensearch-project/OpenSearch/pull/10853))
 - Add back half_float BKD based sort query optimization ([#11024](https://github.com/opensearch-project/OpenSearch/pull/11024))
+- Allowing pipeline processors to access index maping info by passing ingest service ref as part of the processor factory parameters ([#10307](https://github.com/opensearch-project/OpenSearch/pull/10307))
```

Comment:
Do you want to release this in 3.0? Or do you intend to backport to 2.x?

If you plan to backport, please move this down to the Unreleased 2.x section.

Reply:
Yes, it makes sense to backport that. I've made change to the CHANGELOG, can you please help with the backport label, I don't have permissions to add it myself